docs-user/guide-perf-profiling.md

```diff
@@ -4,7 +4,7 @@ Linux has a native profiler called 'perf' that can profile any application. This
 
 [The Firefox Profiler](https://profiler.firefox.com/) provides an alternative UI for these profiles; it knows how to display profiles from perf.
 
-(Importing other profile sources is being investigated, such as in [issue #1138](https://github.com/firefox-devtools/profiler/issues/1138), [PR #1065](https://github.com/firefox-devtools/profiler/pull/1065) and [PR #1260](https://github.com/firefox-devtools/profiler/pull/1260).)
```

Comment:
I feel like PR #1065 should be replaced by issue #1553, what do you think?

Reply:
Done